lib/board.rb

```diff
@@ -3,9 +3,9 @@ class Board
 
   def initialize
     @cells = Hash.new
-    letters = "A".."D"
+    @letters = "A".."D"
```

Comment:
I'm still a little unsure how this range can work if the board size changes, same goes for the numbers. If we can go over a little bit, I'd appreciate it.